Tests/Builders/InteractionTrainingBuilderTests.cs

```
@@ -35,7 +36,7 @@ protected override void InternalSetLocked(bool lockState)
             
             public ISnappableProperty SnappedObject { get; set; }
             
-            public GameObject SnapZoneObject { get; }
+            public ISnapZone SnapZoneObject { get; }
```

Same as above

---

This shows an error for me in Rider but not in Unity and also the Test runs. But it should throw an error, right?